src/ALE/remapping_attic.F90

```diff
@@ -281,7 +287,7 @@ subroutine integrateReconOnInterval( n0, h0, u0, ppoly0_E, ppoly0_coefs, method,
     if ( h0(jL) == 0.0 ) then
       uAve = 0.5 * ( ppoly0_E(jL,1) + ppoly0_E(jL,2) )
     else
-      !### WHY IS THIS NOT WRITTEN AS xi0 = ( xL - x0jLl ) / h0(jL) ---AJA
+      ! WHY IS THIS NOT WRITTEN AS xi0 = ( xL - x0jLl ) / h0(jL) ---AJA
```

Comment:
WHY WAS... _ahem_ ... why was `###` removed?  (Maybe what I mean is, what does `###` mean exactly?)

Reply:
Before we had gitHub issues, `###` was the notation that Alistair and I used to mark things in the MOM6 (and GOLD) code we needed to come back to.  In this case, we are not planning to revisit this expression because it is among the unused code in remapping_attic.F90, but we also don't want to eliminate this comment in case someone does start using this routine again.  You might note that over the past couple of years, most of the lines marked with `###` have been properly fixed.  This is the rare case where we are just going to leave things as they are but remove the equivalent of a private `ToDo` notation.